### PR TITLE
Fix some code oddities

### PR DESCRIPTION
**Pull request by azertyfun**
This PR adds instructions on how to build the database file, updates the home page links (up until now it only had one link to /apps, which doesn't exist), and fixes the test suite failing when the database file has been built.

As discussed on Discord, this PR merges into the `dev` branch because it is ahead of `master`, apart from updates to the README and LICENSE on `master`. You can merge `dev` into master after merging this PR.

### CODE REVIEW

File: boot.go [hunk 1]
```diff
@@ -80,31 +80,31 @@ func BootHandler(w http.ResponseWriter, r *http.Request) {
...
 		return
 	}
-
 	// Make a request to an external server then parse the request
 	req, err := http.Get(request_url)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("Could not contact api:", err)
+	}
+	if req.StatusCode < 200 || req.StatusCode > 300 {
```
Comment (phpeter) [inline boot.go]: Should we instead check if `req.StatusCode > 299` so that we are capturing anything outside of the 2xx/Success range?

**Review by phpeter — APPROVED** [review]

File: boot.go (hunk 1 above)
Comment (sGerli) [reply boot.go]: Yes, you’re right. Change this and I’ll merge.

**Review by sGerli — CHANGES_REQUESTED** [review]
Just change what @phpeter said and I’ll merge

File: boot.go (hunk 1 above)
Comment (azertyfun) [reply boot.go]: Woops, guess I'm dumb. Fixed!

**Review by sGerli — APPROVED** [review]

**Pull request merged**